extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -429,13 +456,10 @@ public void setUp2() throws Exception
 
     SqlTestFramework qf = queryFramework();
 
-    ObjectMapper secondMapper = setupObjectMapper(qf.injector());
-    indexIO = new IndexIO(secondMapper, ColumnConfig.DEFAULT);
-
-    segmentCacheManager = new SegmentCacheManagerFactory(TestIndex.INDEX_IO, secondMapper).manufacturate(newTempFolder("cacheManager"));
-
-    MSQSqlModule sqlModule = new MSQSqlModule();
+    objectMapper = setupObjectMapper(qf.injector());
+    indexIO = new IndexIO(objectMapper, ColumnConfig.DEFAULT);
 
+    segmentCacheManager = new SegmentCacheManagerFactory(TestIndex.INDEX_IO, objectMapper).manufacturate(newTempFolder("cacheManager"));
```

Comment:
Nit: Could we use the above `indexIO` here instead of `TestIndex.INDEX_IO`?

Reply:
Yes, just pushed that.